.github/workflows/build_awshelper.yaml

```diff
@@ -0,0 +1,21 @@
+name: Build awshelper image
+
+on:
+  push:
+    paths:
+    - .github/workflows/build_awshelper.yaml
+    - Docker/awshelper/**
```

Comment:
sorry, late to the game, but I think only triggers awshelper build on these paths are not enough, since there are many jobs that depends on the awshelper image, and the changes to these job can actually happen in locations other than these two. For example https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/jobs/cedar-ingestion-job.yaml#L57

Reply:
I don't think i understand - if you modify `cedar-ingestion-job.yaml` but not the awshelper image, the updated `cedar-ingestion-job` can use awshelper:master without a problem, right? You don't need to use awshelper:<your branch> unless you modified the image?

Reply:
that job uses a file from [here](https://github.com/uc-cdis/cloud-automation/blob/master/files/scripts/healdata/heal-cedar-data-ingest.py) so unless a new awshelper image is being build when changes to that file is merged, then the changes will not always be reflected in the master awshelper image right?

Reply:
oh i see, i'll send another PR

Reply:
thank you!!